### ISSUE TITLE
provision:virtual.

### ISSUE THREAD

**Issue by lukaszachy** [psss/tmt#802]

**Comment by psss** [comment]
Makes sense to me, could be useful, +1 for implementing this. @frantisekz, would you possibly want to have a look a this? Thinking a bit more, for provisioning only you could possibly call `testcloud` command directly. But one has to learn another command line syntax I know...

**Comment by lukaszachy** [comment]
Either `testcloud`or `virsh` directly. However I hope we want tmt be THE tool for testing related stuff. And provisioning machine I can later play with is imho the very first step which is later used with `tmt run provision -h connect`

**Comment by frantisekz** [comment]
This should be really easy to achieve. Basically, it could be achieved with following:

- change connection passed to testcloud objects to variable ( both " connection='qemu:///session' "  occurrences)
- variable would have default 'qemu:///session' value, but should be changeable via some arg to 'qemu:///system'

I can take a look and implement this after PTO, in a week of 20th September.
